src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery shouldn't extends ColorSupplier
we should create object of ColorSupplier and use it
```
ColorSupplier colorSupplier = new ColorSupplier();
colorSupplier.getRandomColor();
```